demo/js/demo.js

```diff
@@ -982,6 +963,16 @@ window.onload = function() {
                 // idx.SQUARE_OF_INCOME
             ],
 
+            backgroundColor: 'white',
+            gridRowProperties: [
```

Comment:
Actually, I named it that because this is not a general implementation. It is _restricted_ to data rows ("grid rows") and is ignored for other subgrids. But your suggestion _is_ in keeping with the way properties are named in general (`backgroundColor` for grid rows, `columnHeaderBackgroundColor` for a column's header cell). **So I will make the change.**